### ISSUE TITLE
Can only access one note in any group via left panel

### ISSUE THREAD

**Issue by taalinukko** [BoostIO/Boostnote#2413]
# Current behavior

Even though there are 9 notes on the screen you can only access one from the left hand panel.  There is no longer a way to select an individual note.

![image](https://user-images.githubusercontent.com/1386789/45906661-966b9600-bdba-11e8-8a79-843e1698fefa.png)

# Expected behavior

I should be able to see and access all the notes via the left hand panel.

# Steps to reproduce

1. Boostnote auto upgraded today
2. Click around test panel and only one note available under each tag.
3. Pressing Ctrl+] still allows navigation to all notes in  a specific tab, so they are not lost

# Environment

- Version : Boostnote 0.11.9
- OS Version and name : Windows 7 Enterprise SP1

**Comment by ZeroX-DG** [comment]
I do not quite understand this, can you provide more info?

**Comment by modmod24** [comment]
his div 'NoteList' is not displayed, as shown in the screenshot.

**Comment by ZeroX-DG** [comment]
I thought he was minimizing the left note list (you can still see the scroll bar of it). Anyway, if you can resize the left note list by hovering the scroll bar on the left and maybe your mouse will change into a resize-arrow and you can resize it. If that doesn't work then maybe you need to re-install the app.

**Comment by taalinukko** [comment]
I tired all the resize arrows, everything I could think of and nothing worked.  

I then did a uninstall/reinstall and that also did not help.

Finally a did an uninstall, then searched the system for anything named Boostnote and then deleted it.  Except for the Boostnote directory in my home directory, I renamed that one.

Then I reinstalled, and things worked!  I then renamed the directory containing my old notes back to Boostnote and everything is good again.

I do not know that the corruption was.  In the event you get something like this again the corrupted file most likely was something stored in AppData that survives an uninstall/reinstall.
